mod-source-record-storage-server/src/main/java/org/folio/rest/impl/SourceStorageMigrationsJobsImpl.java

```diff
@@ -0,0 +1,53 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import org.folio.dataimport.util.ExceptionHelper;
+import org.folio.rest.jaxrs.model.AsyncMigrationJobInitRq;
+import org.folio.rest.jaxrs.resource.SourceStorageMigrationsJobs;
+import org.folio.services.migrations.AsyncMigrationJobService;
+import org.folio.spring.SpringContextUtil;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import javax.ws.rs.core.Response;
+import java.util.Map;
+
+import static java.lang.String.format;
+
+public class SourceStorageMigrationsJobsImpl implements SourceStorageMigrationsJobs {
+
+  public static final String NOT_FOUND_MSG = "Async migration job with id '%s' was not found";
```

Comment:
Does this need to be public?

Reply:
no, just haven't changed this modifier after extraction value to constants by IDE